chrome-helper/eos-google-chrome.py

```diff
@@ -107,12 +107,26 @@ def _get_chrome_flatpak_launcher(self):
 
     parser = argparse.ArgumentParser()
     parser.add_argument('--debug', dest='debug', action='store_true')
+    parser.add_argument('--enable-features', dest='enable_features', default='WebRTCPipeWireCapturer')
+    parser.add_argument('--ozone-platform', dest='ozone_platform')
+    parser.add_argument('--ozone-platform-hint', dest='ozone_platform_hint', default='auto')
 
     parsed_args, otherargs = parser.parse_known_args()
 
     if parsed_args.debug:
         logging.root.setLevel(logging.DEBUG)
 
+    if parsed_args.enable_features:
+        if "WebRTCPipeWireCapturer" in parsed_args.enable_features:
+            otherargs.append(f'--enable-features={parsed_args.enable_features}')
+        else:
+            otherargs.append(f'--enable-features={parsed_args.enable_features},WebRTCPipeWireCapturer')
```

Comment:
Very minor suggestion that leaves space for doing more feature checking in the future.

```
if parsed_args.enable_features:
    if 'WebRTCPipeWireCapturer' not in parsed_args.enable_features:
        parsed_args.enable_features += ',WebRTCPipeWireCapturer'
    otherargs.append(f'--enable-features={parsed_args.enable_features}')
```

To do it even more robustly by using the `,` separator:

```
if parsed_args.enable_features:
    features = parsed_args.enable_features.split(',')
    if 'WebRTCPipeWireCapturer' not in features:
        features.append('WebRTCPipeWireCapturer')
    features = ','.join(features)
    otherargs.append(f'--enable-features={features}')
```

Reply:
Sure!  Updated